docs/configuration/settings.rst

```diff
@@ -0,0 +1,586 @@
+Settings
+########
+
+.. image:: /images/overview-features.png
+    :align: center
+
+Kinto is built to be highly configurable. As a result, the related
+configuration can be verbose, but don't worry, all configuration flags are
+listed below.
+
+.. note::
+
+    In order to ease deployment or testing strategies, *Kinto* reads settings
+    from environment variables, in addition to ``.ini`` files.
+
```

Comment:
I would suggest *explicitly stating the rules* for how environment variables get picked up by Kinto. The example below is good, but I'm totally unclear as to whether it's only variables that begin with `CLIQUET_` or what...